WeakAuras/WeakAuras.lua

```diff
@@ -5450,3 +5452,22 @@ function WeakAuras.FindUnusedId(prefix)
   end
   return id
 end
+
+function WeakAuras.AdvertiseBT2Upgrade()
+  local count = 0
+  for id, aura in pairs(WeakAurasSaved.displays) do
+      if not aura.controlledChildren and aura.triggers then
+          for i, t in pairs(aura.triggers) do
+              if type(i) == "number" then
+                  if t.trigger and t.trigger.type == "aura" then
+                  	count = count + 1
+                  	break
+                  end
+              end
+          end
+      end
+  end
+  if count > 0 then
+    WeakAuras.prettyPrint((L["You have %d auras still using the legacy aura trigger, they are marked with a %s in WeakAuras's Options"]):format(count, WeakAuras.newFeatureString))
```

Comment:
I dislike login nags mostly because I don't think they're effective, not out of any personal dislike. But anyways, you have a typo:
```suggestion
    WeakAuras.prettyPrint((L["You have %d auras still using the legacy aura trigger, they are marked with a %s in WeakAuras Options"]):format(count, WeakAuras.newFeatureString))
```